torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -165,6 +166,14 @@ def _init_streams(
     # Stream for pre-unshard logic, namely allocations and writes for CPU
     # offloading (H2D copy) and mixed precision (low precision cast).
     state._streams["pre_unshard"] = torch.cuda.Stream()
+    # Default stream for computation
+    state._streams["default"] = torch.cuda.current_stream()
+    state._stream_to_name = {
+        torch.cuda.current_stream(): "default",
+        state._streams["unshard"]: "unshard",
+        state._streams["pre_unshard"]: "pre_unshard",
+        state._streams["post_backward"]: "post_backward",
```

Comment:
Is there a way to avoid hardcoding this?

Reply:
Yes, we can do this in a more clever way, but I think it is not a big deal since we have a fixed number of streams. I can make it programmatic in a follow-up if really desired.

Reply:
Note also that this is the only place where we define these streams, so there is no risk of mismatching the `_streams` keys.